src/Components/WebAssembly/Sdk/src/GZipCompress.cs

```diff
@@ -32,26 +32,26 @@ public override bool Execute()
             System.Threading.Tasks.Parallel.For(0, FilesToCompress.Length, i =>
             {
                 var file = FilesToCompress[i];
-                var inputPath = file.ItemSpec;
+                var inputFullPath = file.GetMetadata("FullPath");
```

Comment:
Should we check that Brotli does the same thing?

Reply:
The PR description implies it does:

> The behavior implemented in the changeset mimicks what we already do in the Brotli compression

Also, since we don't do Brotli in development, that probably won't impact this bug.

Reply:
Fair enough, just making sure we cover the corners.

Are there any other instances where we can have this problem? Since this is hard to test automatically.

Reply:
It would be awesome if we did have a test, but that could be very tricky to put together, and I guess we wouldn't want to block this patch on it.

Reply:
Having an automated test for this would be nice but, unfortunately, requires a fair bit of work with our current test infrastructure since we'd have to create a new test project that targets net5 or some other framework add add some logic to modify the project file to target `$(DefaultNetTargetFramework)`. The effort:reward ratio on it is super meh. :/